src/elements/table/extras/th-sortable/ThSortable.js

```diff
@@ -21,7 +21,7 @@ export class ThSortable {
             event.preventDefault();
         }
 
-        if (this.config && this.column && this.config.sorting !== false && this.column.sort !== false) {
+        if (this.config && this.column && this.config.sorting !== false && this.column.sorting !== false) {
```

Comment:
Still want sort here? The sort is 'ASC' or 'DESC', there is already 'sorting' check